### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR adds:
- Issue Templates
- PR template
- Security policy
- labels and its workflow ( in .github and .github/workflows respectively)

- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/documentation.md:1]: THere's a PR open on this one... Either way, though, is fine

**Review by leecalcote — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/documentation.md
Comment (leecalcote) [reply .github/ISSUE_TEMPLATE/documentation.md:1]: It's the issue labels that only you are, specifically empowered to wrangle. 👍

Comment (vishalvivekm) [reply .github/ISSUE_TEMPLATE/documentation.md:1]: Thanks for pointing this out, I removed the commit.

File: .github/ISSUE_TEMPLATE/ci.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<!-- A brief description of what you expected to happen. -->
+
+#### Implementation
+<!-- Specifics on the approach to fulfilling the feature request. -->
+
+#### Acceptance Tests
+<!-- Stipulations of functional behavior or non-functional items that must be in-place in order for the issue to be closed. -->
+
+---
+#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/ci.md:21]: Please copy from layer5io/docs

**Review by jerensl — APPROVED** [review]

**Comment by jerensl** [comment]
@vishalvivekm Looks good to me, nice work

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "Pull Request Labeler"
```
Comment (jerensl) [inline .github/workflows/labeler.yml:1]: Is this duplicate with this PR?
https://github.com/layer5labs/kanvas-snapshot/pull/118/files

**Review by jerensl — APPROVED** [review]

File: .github/workflows/labeler.yml (hunk 2 above)
Comment (vishalvivekm) [reply .github/workflows/labeler.yml:1]: I wasn't aware of this pull request.
I have added the [labeler workflow](https://github.com/layer5labs/kanvas-snapshot/pull/121/files#diff-09b72f3c9a3e4f00ab00cd7000b302db25f056075d8895bd91b3654d6e7e956b) as well as the [labels](https://github.com/layer5labs/kanvas-snapshot/pull/121/files#diff-a22c263686553013feaeb0677d26eeb0b8778a756c4311c1fce13384258026aa) here.

**Pull request merged**